.gitignore

```diff
@@ -0,0 +1 @@
+venv\
```

Comment:
try to use a template one.

check her for example https://github.com/github/gitignore

I think it should be even proposed by github when you start the project

Reply:
Thank you,, it's done